src/Core/Types/Types/Relay/IdFieldValueSerializer.cs

```diff
@@ -129,26 +136,26 @@ private static Type CreateListType(Type elementType)
             return listDefinition.MakeGenericType(typeof(string));
         }
 
-        public object? Serialize(object? value)
+        public virtual object? Serialize(object? value)
```

Comment:
```suggestion
        public object? Serialize(object? value)
```

I don't need this anymore, so can leave it non-virtual.